code/modules/food_and_drink/snacks.dm

```diff
@@ -1309,6 +1362,10 @@
 	heal_amt = 0
 	food_effects = list("food_disease_resist")
 
+/obj/item/reagent_containers/food/snacks/mushroom/terran
```

Comment:
does this get an icon, or different reagents, or anything other than a name change?

Reply:
No, I'll add some static list functionality for ingredients that don't exist "mushrooms" if you think that's preferable

Reply:
Actually I would kind of like to add some `yuck` to them and lower the food quality

Reply:
I'm actually also getting ahold of a sprite

Reply:
They are now sometimes gross, have a color that looks better on pizzas, and sometimes they're gross (low quality).